specs/DisableCrashReporting.md

```diff
@@ -0,0 +1,94 @@
+Process Info
+===
+
+# Background
+Currently, the crash process is created asynchronously with the WebView2 process and listening for any crashes. Once a crash happens, the crash pad process will create a crash dump and send that to Microsoft services for diagnosis. Provide end developer a new API to toggle dump upload consent, once uploads are disabled, dump will still be generated but just not uploaded to Microsoft endpoints. Also, provide an API to get the crash dump folder.
+
+# Examples
+## WinRT and .NET   
+```c#
+
+/// Create WebView Environment with option
+auto options = new CoreWebView2EnvironmentOptions();
+options.DisableCrashReporting = true;
+auto environment = await CoreWebView2Environment.CreateAsync(BrowserExecutableFolder, UserDataFolder, options);
+
+/// Webview creation and setup of completion handler are out of scope of sample
+void WebView_CoreWebView2InitializationCompleted(object sender, CoreWebView2InitializationCompletedEventArgs e)
+{
+    if (e.IsSuccess)
+    {
+        // Get the crash dump folder
+        String crashDumpFolder = webView.CoreWebView2.Environment.CrashDumpFolder();
+    }
+}
+
+```
+## Win32 C++
+```cpp
+void AppWindow::ToggleCrashReportingUploadConsent()
+{
+    // Disable crash report
+    auto options = Microsoft::WRL::Make<CoreWebView2EnvironmentOptions>();
+    CHECK_FAILURE(options->put_DisableCrashReporting(TRUE));
+}
+
+void AppWindow::GetCrashDumpFolder()
+{
+    // Get crashpad folder path  
+    wil::com_ptr<ICoreWebView2Environment11> environment;
+    wil::unique_cotaskmem_string crashPadPath;
+    CHECK_FAILURE(environment->get_CrashDumpFolder(&crashPadPath));
+}
+```
+
+# API Details    
+```
+interface ICoreWebView2Environment11;
+interface ICoreWebView2EnvironmentOptions3;
+
+/// A continuation of the ICoreWebView2Environment interface for
+/// getting the crash dump folder path
+[uuid(F619312E-0399-4520-B700-30818441785A), object, pointer_default(unique)]
+interface ICoreWebView2Environment11 : ICoreWebView2Environment10 {
+  /// Get the crash dump folder that all CoreWebView2's from this
+  /// environment are using.
+  // MSOWNERS: xiaqu@microsoft.com
```

Comment:
It would also be good to say what the crash dump format is. Maybe MSDN already has documentation on this (is it a minidump file? if so https://docs.microsoft.com/en-us/windows/win32/debug/minidump-files)

Reply:
Verified with reliability folks, minidump is the format. Should I document both in `CrashDumpFolderPath` API and `IsCustomCrashReportingEnabled` API?

Reply:
Just the CrashDumpFolderPath where they need to know what is in that folder.